contrib/completion/bash/docker

```diff
@@ -142,13 +146,15 @@ __docker_images() {
 	esac
 
 	local repo_print_command
+	# shellcheck disable=SC2016
```

Comment:
I think [SC2016](https://github.com/koalaman/shellcheck/wiki/SC2016) is so basic that we should not warn about it. 
We have several computed awk expressions that all produce these warnings. Please add SC2016 to the general exclusions.